conf/coverage.sh

```diff
@@ -54,6 +54,8 @@ $MPIEXEC -n 1 $PYTHON -m coverage run -m mpi4py -c "import sys; sys.exit('error'
 $MPIEXEC -n 1 $PYTHON -m coverage run -m mpi4py -c "from mpi4py import MPI; 1/0;"  > /dev/null 2>&1 || true
 
 $MPIEXEC -n 1 $PYTHON -m coverage run test/test_util_pkl5.py  -q 2> /dev/null
+$MPIEXEC -n 2 $PYTHON -m coverage run test/test_util_pkl5.py  -q 2> /dev/null
+$MPIEXEC -n 3 $PYTHON -m coverage run test/test_util_pkl5.py  -q 2> /dev/null
```

Comment:
Just curious: `-n 2` is not enough?

Reply:
No, if you want to test `bcast()` on intercommunicators, and exercise both the `MPI.ROOT` and `MPI.PROC_NULL` branches. So you need a universe of 2+1 processes to be rearranged as a 2<->1 intercommunicator. So 3 processes is the minimum required to achieve full test coverage.

Reply:
Thanks for explanation, Lisandro. Yeah it was a silly question if I had read the `testBcastInter` test 😅